### DISCUSSION TITLE
Unable to set aad is none

### DISCUSSION THREAD

**Discussion by RaveenRasaiya** [latchset/jwcrypto#233] [Q&A]
Hi All,

Thank you for great library and  I am trying to create JWE payload for robot test using below code,

  protected_header = {
            "alg":"RSA1_5",            
            "enc":"A128GCM"
        }
        jwetoken = jwe.JWE(payload.encode('utf-8'),
                           recipient=public_key,                           
                           protected=protected_header)
        enc = jwetoken.serialize(compact=True)

I want  to set aad should be None . Why it is not supported

Thanks

**Comment by simo5** [comment]
aad is None by default, there is nothing you need to do.

**Reply by simo5** [reply]
Can you show with an example what is your issue? (Ie what you get and what you expect instead?)
